debian/mgmt.service

```diff
@@ -0,0 +1 @@
+../misc/mgmt.service
```

Comment:
Ok, apparently I can't read :)

I'll see how it is done in Debian and do the same thing. Personally, I don't see any point in manually installing and then explicitly calling some dh_* when it can be automagically handled with one symlink.